spinalcordtoolbox/reports/qc.py

```diff
@@ -840,6 +840,12 @@ def qcslice_layout(x): return x.single()
         qcslice_type = qcslice.Sagittal([Image(fname) for fname in fname_list], p_resample=None)
         qcslice_operations = [QcImage.smooth_centerline, QcImage.highlight_pmj, QcImage.listed_seg]
         def qcslice_layout(x): return x.single()
+    elif process in ['sct_image -stitch']:
+        plane = 'Sagittal'
+        dpi = 150
```

Comment:
I'm not 100% sure about this DPI setting. I experimented with both `100` and `300`, too? But, depending on the subject/image, this can look too big or too small. It can be tricky to balance A) being able to see individual voxel details versus B) being able to fit most/all of the scan on the screen.

The trouble here is that `sct_image -stitch` often will be working with images that are much larger than typical for SCT (e.g. full-body scans). Because of that, it's hard to fit everything on a typical 1920x1080 screen; settings that will look good for a torso scan may look poor for a full-body scan.

@jqmcginnis If you would like, feel free to play with `dpi` as you try out this feature. :)

Reply:
@joshuacwnewton Thank you very much for the info! I will be processing a small cohort over the weekend, and will update you which setting works best for roughly ~30-50 patients. My feeling is that most SCT users will use this feature for whole spine stitching (and not full body scans), so that should probably be our (best / initial) assumption? :slightly_smiling_face: